spec/models/outgoing_message_spec.rb

```diff
@@ -1789,6 +1789,29 @@
 
   end
 
+  describe '#record_email_failure' do
+    let(:outgoing_message) { FactoryBot.create(:initial_request) }
+
+    it 'sets the status to "failed"' do
+      outgoing_message.record_email_failure('test')
```

Comment:
Would be worth making this a little clearer as to what ends up being passed as an argument IRL.

Maybe `'Exception#message'`?